pallets/farming/src/lib.rs

```diff
@@ -388,6 +407,34 @@ pub mod pallet {
     #[pallet::storage]
     pub type PoolFarmers<T: Config> =
         StorageMap<_, Identity, T::AccountId, Vec<PoolFarmer<T>>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn lp_min_xor_for_bonus_reward)]
+    pub type LpMinXorForBonusReward<T: Config> =
+        StorageValue<_, Balance, ValueQuery, DefaultLpMinXorForBonusReward<T>>;
+
+    #[pallet::call]
+    impl<T: Config> Pallet<T> {
+        #[pallet::call_index(0)]
+        #[pallet::weight(<T as Config>::WeightInfo::set_lp_min_xor_for_bonus_reward())]
+        pub fn set_lp_min_xor_for_bonus_reward(
+            origin: OriginFor<T>,
+            new_lp_min_xor_for_bonus_reward: Balance,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+            ensure!(
+                new_lp_min_xor_for_bonus_reward >= balance!(1),
```

Comment:
Balance is u128 which is always >= 0

Reply:
Ok, I clarified just to not miss the error.